services/vue/src/components/public/templates/TemplateList.vue

```diff
@@ -410,6 +425,22 @@ export default defineComponent({
       (this.$refs.dialogCreate as InstanceType<typeof TemplateDialogCreate>)
         ?.openFromTemplate(template);
     },
+    async toggleHideTemplate({ id }: templates.Template) {
+      if (!this.perms.update) {
+        return;
+      }
+
+      this.loading = true;
+      const { content: template } = await this.$ezReeport.sdk.templates.getTemplate(id);
+      await this.$ezReeport.sdk.templates.upsertTemplate({
+        id,
+        hidden: !template.hidden,
+        name: template.name,
+        body: template.body,
+        tags: template.tags,
+      });
+      this.fetch();
+    },
```

Comment:
If one of the promises rejects, user may experience infinite loading without error feedback